x/wasm/keeper/keeper_test.go

```diff
@@ -1046,6 +1055,47 @@ func TestMigrateWithDispatchedMessage(t *testing.T) {
 	assert.Equal(t, deposit, balance)
 }
 
+func TestIterateContractsByCode(t *testing.T) {
```

Comment:
I would extend this one (or make another one) that has a migrate - where a few contracts have different orders.

Migrating contract 1 to the same hackatom code (which we support) would change the order for example. 
Actually, better a second test - this is a great one to show creation usage - which must work for non-migrated contracts - we need another one for ordering of migrated contracts

Reply:
👌 I will add a new one.